tools/donate-cpu.py

```diff
@@ -126,7 +126,12 @@ def removeTree(folderName):
 
 
 def wget(url, destfile):
-    os.remove(destfile)
+    if os.path.exists(destfile):
+        if os.path.isfile(destfile):
+            os.remove(destfile)
+        else:
+            print('Error: ' + destfile + ' exists but it is not a file! Please check the path and delete it manually.')
+            return False
```

Comment:
If it's a folder the rmtree could be used.

Reply:
yes.. that is true. Then I would suggest `sys.exit(1)` so the script terminates.

Reply:
Ok, makes sense.